scss/layout/_home.scss

```
@@ -0,0 +1,57 @@
+.home {
+  .banner {
+    background-color: $bg-gray;
+
+    &-wrapper {
+      @include flex($justify: space-between, $align: center, $wrap: nowrap);
+    }
+
+    &__left {
+      margin-top: 40px;
+      @include flex($justify: center, $align: center, $direction: column);
+      text-align: center;
+      @media only screen and (max-width: 768px) {
+        margin-top: 0px;
+      }
+    }
+
+    &__product {
+        @media only screen and (max-width: 768px) {
+            height: 282px;
+          }
+    }
+
+    &__text {
+      font-size: $fs-xl;
+      color: $text-other;
+      margin-top: 5px;
+    }
+
+    &__price {
+      font-size: $fs-9xl;
+      color: $text-green;
+      font-weight: 300;
+      margin-top: 5px;
+    }
+
+    &__buy {
+      text-decoration: underline;
+      font-size: $fs-l;
+      margin-top: 10px;
+    }
+
+    &__right {
+      height: 856px;
+      @include flex($justify: center, $align: flex-end);
+      img {
```

```suggestion
    &__right {
      @include flex($justify: center, $align: flex-end);
       height: 856px;
       
      img {
```
Mỗi block cách nhau 1 hàng nhé, với lại xem sử dụng các đơn vị tương đối như %, vw, vh thay vì giá trị tuyệt đối như px, em, rem như này nhé